time/src/lib.rs

```diff
@@ -0,0 +1,86 @@
+//! LV2 specification to describe position in time and passage of time, in both real and musical terms.
+//!
+//! The original [specification](https://lv2plug.in/ns/ext/time/time.html) contains means to describe time for LV2 values in RDF files. This implementation is focused on the stock time descriptions defined by the specification by binding them to marker types.
+extern crate lv2_core as core;
+extern crate lv2_sys as sys;
+extern crate lv2_urid as urid;
+
+use urid::prelude::*;
+
+/// All time properties  URI bounds
+pub mod time {
+    use core::UriBound;
+
+    pub struct Bar;
+    unsafe impl UriBound for Bar {
+        const URI: &'static [u8] = sys::LV2_TIME__bar;
+    }
+
+    pub struct BarBeat;
+    unsafe impl UriBound for BarBeat {
+        const URI: &'static [u8] = sys::LV2_TIME__barBeat;
+    }
+
+    pub struct Beat;
+    unsafe impl UriBound for Beat {
+        const URI: &'static [u8] = sys::LV2_TIME__beat;
+    }
+
+    pub struct BeatUnit;
+    unsafe impl UriBound for BeatUnit {
+        const URI: &'static [u8] = sys::LV2_TIME__beatUnit;
+    }
+
+    pub struct BeatsPerBar;
+    unsafe impl UriBound for BeatsPerBar {
+        const URI: &'static [u8] = sys::LV2_TIME__beatsPerBar;
+    }
+
+    pub struct BeatsPerMinute;
+    unsafe impl UriBound for BeatsPerMinute {
+        const URI: &'static [u8] = sys::LV2_TIME__beatsPerMinute;
+    }
+
+    pub struct Frame;
+    unsafe impl UriBound for Frame {
+        const URI: &'static [u8] = sys::LV2_TIME__frame;
+    }
+
+    pub struct FramesPerSecond;
+    unsafe impl UriBound for FramesPerSecond {
+        const URI: &'static [u8] = sys::LV2_TIME__framesPerSecond;
+    }
+
+    pub struct Speed;
+    unsafe impl UriBound for Speed {
+        const URI: &'static [u8] = sys::LV2_TIME__speed;
+    }
+
+    pub struct Position;
+    unsafe impl UriBound for Position {
+        const URI: &'static [u8] = sys::LV2_TIME__position;
+    }
+}
```

Comment:
You're missing the Time and Rate URIs here.

Reply:
I only put URIs that correspond to time properties, in same way lv2_units use only units instance. To declare all URIs we need to think about new convention of naming and/or scoping because there is some names differ by letter case. Seems "Class" always with an upper case letter and others start by a lower case letter.

Reply:
Yes, that's right. However, I would still prefer to have all URIs in there. Maybe you can pre- or append "Class" to the names or something like that.